samples/tex-string.js

```diff
@@ -12,8 +13,7 @@ let html = MathJax.document('<html></html>', {
 
 MathJax.handleRetriesFor(() => {
 
-    html.TestMath(process.argv[3] || '').compile();
-    let math = html.math.pop().root;
+    let math = html.convert(process.argv[3] || '', {end: STATE.TYPESET});
```

Comment:
I think the state here should be `STATE.COMPILED` (or `STATE.CONVERT` to be consistent with the other ones).